java-tools/OsmAndServer/src/main/java/net/osmand/server/WebSecurityConfiguration.java

```diff
@@ -49,68 +47,45 @@
 import org.springframework.web.cors.CorsConfigurationSource;
 import org.springframework.web.cors.UrlBasedCorsConfigurationSource;
 
-import net.osmand.server.api.repo.CloudUserDevicesRepository;
-import net.osmand.server.api.repo.CloudUserDevicesRepository.CloudUserDevice;
-import net.osmand.server.api.repo.CloudUsersRepository;
-import net.osmand.server.api.repo.CloudUsersRepository.CloudUser;
-import net.osmand.server.api.services.UserdataService;
+import java.io.IOException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.util.*;
+import java.util.concurrent.ConcurrentHashMap;
 
 @Configuration
 @EnableOAuth2Client
 public class WebSecurityConfiguration {
-	
-	protected static final Log LOG = LogFactory.getLog(WebSecurityConfiguration.class);
+
 	public static final String ROLE_PRO_USER = "ROLE_PRO_USER";
 	public static final String ROLE_ADMIN = "ROLE_ADMIN";
 	public static final String ROLE_SUPPORT = "ROLE_SUPPORT";
 	public static final String ROLE_USER = "ROLE_USER";
+	protected static final Log LOG = LogFactory.getLog(WebSecurityConfiguration.class);
 	private static final int SESSION_TTL_SECONDS = 3600 * 24 * 30;
-    
```

Comment:
Why is everything in the class reordered? Stick to the order: bean injections first, then constants, classes, and only after that the rest of the code